modules/oe_translation_remote/src/TranslationRequestRemoteInterface.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_translation_remote;
+
+use Drupal\oe_translation\Entity\TranslationRequestInterface;
+use Drupal\oe_translation\LanguageWithStatus;
+use Drupal\oe_translation_remote\Entity\RemoteTranslatorProviderInterface;
+
+/**
+ * Interface for remote translation request entities.
+ */
+interface TranslationRequestRemoteInterface extends TranslationRequestInterface {
```

Comment:
This is the interface the remote bundle classes of TranslationRequest will implement (using the RemoteTranslationRequestEntityTrait)